src/components/header/dropdown-company/dropdown-company.scss

```diff
@@ -0,0 +1,82 @@
+*{
+    text-decoration: none;
+    bottom: unset;
+    color: #fff;
+}
+.button-company{
+    padding: 10px 7px;
+    display: flex;
+    background-color: unset;
+    color: #fff;
+    border: unset;
+    font-size: 16px;
+}
+.svg-company{
+    display: flex;
+    align-items: center;
+    margin-left: 9px;
+}
+.content-dropdown{
+    display: flex;
+    align-items: center;
+    color: #000 !important;
+    font-family: "Helvetica Neue", sans-serif;
```

Comment:
You can add the font-family in the `body *` instead of use it directly in a class